installer/pkg/common/constants.go

```diff
@@ -34,7 +34,7 @@ const (
 	RegistryAuthSecret          = "builtin-registry-auth"
 	RegistryTLSCertSecret       = "builtin-registry-certs"
 	RegistryFacadeComponent     = "registry-facade"
-	RegistryFacadeServicePort   = 3000
+	RegistryFacadeServicePort   = 30000
```

Comment:
that value could produce some collision (default [initial value](https://kubernetes.io/es/docs/concepts/services-networking/service/#nodeport) for node ports)

Reply:
Yes it could. @kylos101 has done some post-processing for Core Dev on daemonsets to avoid collisions as part of his Werft job. Is that the collision you're talking about?

Reply:
The range core-dev uses for [registry-facade is 30000-31000](https://github.com/gitpod-io/gitpod/blob/main/.werft/build.ts#L300). @aledbf is that the potential collision you were referring? If yes, I think we'll be okay, because this is referring registry-facade.

Reply:
Yes. Are we doing the same post-processing on user clusters?

Reply:
No we're not. The general principle that we're working on is one deployment of Gitpod per cluster. It's not explicitly noted at the moment, but it's one that will be documented when we do the public introductory docs